tests/parser/extract-queries.php

```diff
@@ -0,0 +1,349 @@
+<?php
+
+/**
+ * This script scans MySQL test files and extracts SQL queries from them.
+ *
+ * The tests are written using The MySQL Test Framework:
+ *   https://dev.mysql.com/doc/dev/mysql-server/latest/PAGE_MYSQL_TEST_RUN.html
+ *
+ * See alos the mysqltest Language Reference:
```

Comment:
Nitpick:

```suggestion
 * See also the mysqltest Language Reference:
```

Otherwise it's a great comment and really helps understand the context of this file. Thank you!